jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -851,6 +851,19 @@ private native void copySwapMemory0(Object srcObj, long srcOffset, Object destOb
 	 * @throws IllegalArgumentException if field is static
 	 */
 	private native long objectFieldOffset0(Field field);
+	
+/*[IF Java10]*/
+	/*
+	 * Returns byte offset to field.
+	 * 
+	 * @param class with desired field
+	 * @param string name of desired field
+	 * @return offset to start of class or interface
+	 * 
+	 * @throws IllegalArgumentException if field is static
+	 */
+	private native long objectFieldOffset1(Class c, String fieldName);
```

Comment:
Where do I find the implementation of `objectFieldOffset1`?  Is that in a separate PR or is this intended to be a stub?

Reply:
It was written in by @JasonFengJ9 in sun_misc_Unsafe.cpp for initial java 10 support. Thanks Jason :)